ui/webpack.config.js

```diff
@@ -17,241 +17,243 @@ const eslintConfig = {
-      'X-Frame-Options': 'SAMEORIGIN',
-      'X-Content-Type-Options': 'nosniff',
-      'X-XSS-Protection': '1; mode=block',
-      'Referrer-Policy': 'no-referrer',
-      'Strict-Transport-Security':
-        'max-age=31536000; includeSubDomains; always;',
+    resolve: {
+      alias: {
+        'react-native': 'react-native-web',
+        'react-dom': '@hot-loader/react-dom',
+      },
+      extensions: [
+        '.web.jsx',
+        '.web.js',
+        '.wasm',
+        '.mjs',
+        '.jsx',
+        '.js',
+        '.json',
+      ],
     },
-  },
-  module: {
-    rules: [
-      {
-        test: /\.html$/,
-        use: [
-          {
-            loader: 'html-loader',
-            options: {
-              attrs: ['img:src', 'link:href'],
-            },
-          },
-        ],
+    devServer: {
+      port,
+      hot: true,
+      historyApiFallback: {
+        disableDotRule: true,
+      },
+      overlay: true,
+      stats: {
+        all: false,
+        errors: true,
+        timings: true,
+        warnings: true,
       },
-      {
-        include: [`${__dirname}/src`, `${__dirname}/test`],
-        test: /\.(js|jsx)$/,
-        use: [
-          {
-            loader: 'babel-loader',
-            options: {
-              cacheDirectory: true,
-              babelrc: false,
-              configFile: false,
-              presets: [
-                [
-                  '@babel/preset-env',
-                  {
-                    debug: false,
-                    useBuiltIns: false,
-                    shippedProposals: true,
-                    targets: {
-                      browsers: [
-                        'last 2 Chrome versions',
-                        'last 2 Firefox versions',
-                        'last 2 Edge versions',
-                        'last 2 Opera versions',
-                        'last 2 Safari versions',
-                        'last 2 iOS versions',
-                      ],
+      host: process.env.HOST || DEFAULT_HOST,
+      https: true,
+      headers: {
+        'Content-Security-Policy':
+          "default-src 'none'; script-src 'self' 'unsafe-eval'; img-src 'self' https://*.gravatar.com https://*.githubusercontent.com https://i1.wp.com; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com; font-src https://fonts.gstatic.com; connect-src https://localhost:8010 'self' https://balrog-localdev.auth0.com https://www.googleapis.com/; frame-src https://balrog-localdev.auth0.com; frame-ancestors 'none'; base-uri 'none'; form-action 'none'",
```

Comment:
I see `'unsafe-eval'` here. I wonder why this is an issue now. It is what Neutrino was generating...